app/dashboard/views.py

```diff
@@ -776,3 +782,75 @@ def toolbox(request):
+
+            # Instantiate Colorado Coin contract
+            contract = w3.eth.contract(coin.contract_address, abi=abi)
+
+            tx = contract.functions.transfer(address, coin.amount * 10**16).buildTransaction({
+                'nonce': w3.eth.getTransactionCount(settings.COLO_ACCOUNT_ADDRESS),
+                'gas': 40000,
+                'gasPrice': recommend_min_gas_price_to_confirm_in_time(5) * 10**9
+            })
+
+            signed = w3.eth.account.signTransaction(tx, settings.COLO_ACCOUNT_PRIVATE_KEY)
+            transaction_id = w3.eth.sendRawTransaction(signed.rawTransaction)
```

Comment:
You could `transaction_id = w3.eth.sendRawTransaction(signed.rawTransaction).hex()` and reference `transaction_id` in place of calling `transaction_id.hex()` throughout.